src/webgpu/api/operation/shader_module/compilation_info.spec.ts

```diff
@@ -76,13 +112,18 @@ g.test('compilationInfo_returns')
     - Test that the compilation info for valid shader modules contains no errors.
     - Test that the compilation info for invalid shader modules contains at least one error.`
```

Comment:
nit. Update descriptions here (and in other tests) to indicate that we are testing with sourcemaps to ensure that there are no crashes. Might be helpful to indicate that they don't really do anything atm also!